packages/components/alert/src/components/alert-description/AlertDescription.tsx

```diff
@@ -0,0 +1,36 @@
+import { Typography } from '@julo-ui/typography';
+import { forwardRef, cx } from '@julo-ui/system';
+
+import { useAlertContext } from '../../AlertProvider';
+
+import { alertDescription } from './styles';
+import type { AlertDescriptionProps } from './types';
+
+const AlertDescription = forwardRef<AlertDescriptionProps, 'div'>(
+  (props, ref) => {
+    const { children, className, sx, ...resProps } = props;
+    const alertContext = useAlertContext();
+
+    if (!alertContext) throw new Error('AlertIcon should be within Alert');
```

Comment:
Please adjust this code. The latest update of @julo-ui/context already have an enhancement of the type guard so you don't need to check if you don't change the `strict` to `false` from createContext